### PR TITLE
Linters configuration

### PR DESCRIPTION
**Pull request by AleMdta**
In this milestone we:
-Configure linters
-Add a readme file 
-Use GitHub flow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,145 +90,52 @@ Example commands:
...
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+- GitHub: [@AleMdta](https://github.com/AleMdta)
+- Twitter: [@AleMdta](https://twitter.com/AleMdta)
+- LinkedIn: [Alejandra Mendieta](https://www.linkedin.com/in/alejandra-mendieta-75480626a)
 
-<!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
 > Describe 1 - 3 features you will add to the project.
```
Comment (ademibowale) [inline README.md:109]: - [ ] Kindly remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with your custom content.

File: README.md [hunk 2]
```diff
@@ -73,49 +43,30 @@ After you're finished please remove all the comments and instructions!
...
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
```
Comment (ademibowale) [inline README.md:55]: - [ ] Remove the place holder texts and Make sure the you have added the section Future Features, including 1-3 potential features to be added e.g HTNL .CSS and LINTERS

File: README.md [hunk 3]
```diff
@@ -144,145 +90,52 @@ Example commands:
...
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```
Comment (ademibowale) [inline README.md:137]: - [ ] Remove the placeholder text as its just use to describe how to modifythe templete.

File: README.md [hunk 4]
```diff
@@ -144,145 +90,52 @@ Example commands:
 
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone git@github.com: https://github.com:AleMdta/hello-microverse.git
```
Comment (ademibowale) [inline README.md]: - [ ]  Kindly narate the Setup of the project explain process how to clone your project repo

Installalation:

Usage:

Run tests:

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @AleMdta , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._
- [ ] kindly make your pull Request summary to be more descriptive ,describing introduced changes in more detail.  
- [ ] I would kindly suggest you to edit your PR tittle  to be more descript e.g `HELLO MICROVERSE`


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @AleMdta , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AleMdta** [comment]
In this milestone:
Change information in the readme file

**Review by jmonto55 — APPROVED** [review]
# STATUS: APPROVED 🟢

Hi @AleMdta , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

AgreeSealGIF



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**